app/models/video.py

```diff
@@ -0,0 +1,40 @@
+from flask import current_app
+from app import db
+from datetime import datetime
+
+class Video(db.Model):
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)  
+    title = db.Column(db.String) 
+    release_date = db.Column(db.DateTime, nullable=False)
+    total_inventory = db.Column(db.Integer, nullable=False)
+    available_inventory = db.Column(db.Integer, nullable=False) 
```

Comment:
Like with `videos_checked_out_count`, consider how you might calculate this value rather than storing it as an attribute in the database.